docker-compose.yml

```diff
@@ -22,6 +22,31 @@ services:
       - "8080:8080"
       - "8081:8081"
 
+  db:
+    image: postgres:latest
+    networks:
+      - kafka_network
+    environment:
+      POSTGRES_DB: mydb
+      POSTGRES_USER: username
+      POSTGRES_PASSWORD: password
+    volumes:
+      - ./db/data:/var/lib/postgresql/data
```

Comment:
Does this volume need to be externally available? We would prefer its internal volume if not.

Reply:
I will put it in gitignore, will this solve the problem?

Reply:
I don't we will need to access these files. Please make a volume, instead of mapping the directory to host: https://docs.docker.com/storage/volumes/#use-a-volume-with-docker-compose

Reply:
FYI, docker compose will persist these files even after `docker compose down` to the next `docker compose up` unless you explicitly give a flag to remove volumes too: `docker compose down --volumes`.